generators/java/sdk/versions.yml

```diff
@@ -1,10 +1,19 @@
+- changelogEntry:
+    - summary: |
+        We now provide endpoint methods for streaming byte array requests in addition to the previous methods accepting
+        byte array directly.
+      type: feat
+  createdAt: '2024-09-27'
+  irVersion: 46
+  version: 2.3.0
+
 - changelogEntry:
     - summary: |
         Bump Jackson version to latest (2.17.2)
       type: chore
   createdAt: '2024-09-26'
   irVersion: 46
-  version: 1.2.0
+  version: 2.2.0
```

Comment:
this was a blunder 😅 

thankfully, we never previously released a `1.2.0` and niels won't be using that until next week
I deleted this erroneous `1.2.0` release from docker hub